windows/src/desktop/kmshell/xml/strings.xml

```diff
@@ -1,405 +1,1187 @@
-  <string name="S_Toolbar_ViewFontHelper" comment="Toolbar Context Help/PlainText: Toolbar help - View Font Helper pop-up text (introduced 7.0.230.0)">View Font Helper</string>
-  <string name="S_Toolbar_ViewCharacterMap" comment="Toolbar Context Help/PlainText: Toolbar help - View Character Map pop-up text (introduced 7.0.230.0)">View Character Map</string>
-  <string name="S_Toolbar_OpenConfiguration" comment="Toolbar Context Help/PlainText: Toolbar help - Open Keyman Configuration pop-up text (introduced 7.0.230.0)">Open Keyman Configuration</string>
-  <string name="S_Toolbar_OpenHelp" comment="Toolbar Context Help/PlainText: Toolbar help - Open help pop-up text (introduced 7.0.230.0)">Open Help</string>
-  <string name="S_Toolbar_CloseOnScreenKeyboard" comment="Toolbar Context Help/PlainText: Toolbar help - Close OSK pop-up text (introduced 7.0.230.0)">Close On Screen Keyboard</string>
-
-  <!-- - - - - - - - - - - - - - - - - - -->
-  <!-- Menu Items                        -->
-  <!-- - - - - - - - - - - - - - - - - - -->
-
-  <string name="S_Menu_SwitchKeymanOff" comment="System Tray Menu Items/PlainText: Systray item - Switch Keyman Off [&amp; precedes alt + access-character] (introduced 7.0.230.0)">Switch Keyman &amp;Off</string>
-  <string name="S_Menu_OnScreenKeyboard" comment="System Tray Menu Items/PlainText: Systray item - OSK [&amp; precedes alt + access-character] (introduced 7.0.230.0)">On Screen &amp;Keyboard</string>
-  <string name="S_Menu_KeyboardUsage" comment="System Tray Menu Items/PlainText: Systray item - Keyboard Usage [&amp; precedes alt + access-character] (introduced 7.0.230.0)">Keyboard &amp;Usage</string>
-  <string name="S_Menu_FontHelper" comment="System Tray Menu Items/PlainText: Systray item - Font Helper [&amp; precedes alt + access-character] (introduced 7.0.230.0)">&amp;Font Helper</string>
-  <string name="S_Menu_CharacterMap" comment="System Tray Menu Items/PlainText: Systray item - Character Map [&amp; precedes alt + access-character] (introduced 7.0.230.0)">Character &amp;Map</string>
-  <string name="S_Menu_TextEditor" comment="System Tray Menu Items/PlainText: Systray item - Text Editor [&amp; precedes alt + access-character] (introduced 7.0.230.0)">&amp;Text Editor...</string>
-  <string name="S_Menu_Configuration" comment="System Tray Menu Items/PlainText: Systray item - Configuration [&amp; precedes alt + access-character] (introduced 7.0.230.0)">&amp;Configuration...</string>
-
-  <string name="S_Menu_Help" comment="System Tray Menu Items/PlainText: Systray item - Help [&amp; precedes alt + access-character] (introduced 7.0.230.0)">&amp;Help...</string>
-  <string name="S_Menu_Exit" comment="System Tray Menu Items/PlainText: Systray item - Exit [&amp; precedes alt + access-character] (introduced 7.0.230.0)">E&amp;xit</string>
+%1:s</string>

```

Comment:
Is this newline intentional?

Reply:
Yes, `%1:s` is the list of fonts which will also be uninstalled. It's a little weak in terms of localization but not a big deal.